x-pack/plugins/index_lifecycle_management/public/extend_index_management/components/index_lifecycle_summary.js

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component, Fragment } from 'react';
+import moment from 'moment-timezone';
+import {
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiDescriptionListDescription,
+  EuiSpacer,
+  EuiTitle,
+  EuiLink,
+  EuiPopover,
+  EuiPopoverTitle,
+} from '@elastic/eui';
+import { i18n }  from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { getPolicyPath } from '../../services/navigation';
+const HEADERS = {
+  policy: i18n.translate('xpack.indexLifecycleMgmt.summary.headers.lifecyclePolicyHeader', {
+    defaultMessage: 'Lifecycle policy',
+  }),
+  phase: i18n.translate('xpack.indexLifecycleMgmt.summary.headers.currentPhaseHeader', {
```

Comment:
The ids here are inconsistent. Some read `xpack.indexLifecycleMgmt.summary` while others read `xpack.idxMgmt.indexLifecycleMgmtSummary`

Reply:
Done, went with xpack.idxMgmt.indexLifecycleMgmtSummary